Interfaces/GCClassic/main.F90

```diff
@@ -1891,24 +1891,48 @@ PROGRAM GEOS_Chem
           ! CH4 columns from the GOSAT instrument
           IF ( Input_Opt%GOSAT_CH4_OBS ) THEN
              IF ( ITS_A_NEW_HOUR() ) THEN
+                IF ( Input_Opt%useTimers ) THEN
```

Comment:
There are a bunch of if statements here. I wonder if this would be more efficient with a new logical that checks if any of the CH4 columns will be computed, and also embed within its a new hour:
```
IF ( Input_Opt%satellite_CH4_columns ) THEN
   IF ( ITS_A_NEW_HOUR ) THEN
         (Start timer here)
        IF ( Input_Opt%GOSAT_CH4_OBS) THEN
             ...
        ENDIF
        IF ( Input_Opt%AIRS_CH4_OBS) THEN
             ...
        ENDIF
        IF ( Input_Opt%TCCON_CH4_OBS) THEN
             ...
        ENDIF
        (End timer here)
    ENDIF
ENDIF
```

Reply:
Nice catch @lizziel!  I've implemented this in commit 48860276.